torchao/quantization/quant_api.py

```diff
@@ -2409,8 +2405,16 @@ def _module_fqn_to_config_handler(
         # Maybe: we can add module type specific config in the future, in needed
         c = config.module_fqn_to_config[module_fqn]
```

Comment:
nit: assert that the pattern does not start with `re:`, for clarity

Reply:
I can update this in my PR.